docs/using-orbs.md

```diff
@@ -0,0 +1,39 @@
+# Using Orbs in CirlceCI Configuration
+
+Orbs are packages of CircleCI configuration shared across projects. Orbs are made available for use in configuration through the `orbs` key in the top level of your config.yml. 
+
+Importing a set of orbs might look like:
+
+```
+orbs:
+  rails: circleci/rails@1.13.3
+  python: circleci/python@2.1
+  heroku: circleci/heroku@latest
+```
+
+The above would make three orbs available, one for each key in the map and named the same as the keys in the map. 
+
+You can also write [inline orbs](inline-orbs), which can be especially useful during development of a new orb. Because the values of the above keys under `orbs` are all scalar values they are assumed to imports based on the orb URI format.
+
+## Orb URI Format
+Orb URIs have the format:
+
+`[namespace]/[orb]@[version]`
+
+Orb namespaces may have restrictions that prevent you from accesing orbs in those namespaces based on whether the build and the scope of the namespace are congruent.
+
+## Semantic Versioning in Orbs
+
+Orbs are published with the standard 3-number [semantic versioning system](https://semver.org/), `major.minor.patch`, and orb authors need to adhere to semantic versioning. Within `config.yml`, you can specify wildcard version ranges to resovle orbs. You can also use the special string `latest` to pull in whatever the highest version number is.  
+
+For instance, the versions below have the following meaning:
+
+1. `circleci/python@latest` - use the latest version of the Python orb in the registry at the time a build is triggered.
```

Comment:
Maybe above we could go into more detail about when exactly these are resolved, and how it's different for orbs and for configs? Something like "Orb dependencies *inside an orb* are resolved and frozen at the time the orb is published. Orb dependencies inside a user's build configuration are resolved every time the build is triggered."

Reply:
Agree -- config time resolution vs. publish time resolution.